inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -376,6 +378,11 @@ void GNADeviceHelper::close() {
     GNADeviceClose(nGNAHandle);
     nGNAHandle = 0;
 #else
+    if (wasAnyRequestScheduled) {
+        try {
+            wait(lastSheduledRequestId);
```

Comment:
this should not be called if Gna2RequestWait was already called and got status of completition.

Reply:
are there any issues with it (except exception?) keeping track of all calls may waste memory

Reply:
Or actually i can track only single wait

Reply:
New implementation takes different approach, which solves this issue